llvm/lib/Support/Path.cpp

```diff
@@ -711,6 +711,13 @@ StringRef remove_leading_dotslash(StringRef Path, Style style) {
   return Path;
 }
 
+StringRef remove_leading_dotbackslash(StringRef Path) {
+  // Remove leading ".\\".
+  if (Path.size() > 2 && Path[0] == '.' && Path[1] == '\\')
+    Path = Path.substr(2);
+  return Path;
+}
+
```

Comment:
Instead of a new routine, can ``remove_leading_dotslash`` be reused with the appropriate ``Style``?

Reply:
It's not the `Style` argument that's the issue, it's the call to `is_separator` that would make the change more complicated than it should be. Furthermore, we only need to check if we have the `.\\` at the beginning of the string, no need to visit the entire string which is what `remove_leading_dotslash` is doing. 
But I can make the change if you don't want to see this additional function.

Reply:
I see.   You specifically don't want the loop?  In that case, I am fine with your new function - except maybe give it a distinct name, so the leading vs all distinction is clear.

How about something like: ``remove_leading_dotbackslash_only``?   Just a suggestion.  Please pick a spelling that sounds good to you.

Reply:
Sure. I can name it that way. Thanks.